components/inputs/input-date-range.js

```diff
@@ -145,6 +145,21 @@ class InputDateRange extends FormElementMixin(RtlMixin(LocalizeCoreElement(LitEl
 		`;
 	}
 
+	updated(changedProperties) {
+		super.updated(changedProperties);
+
+		changedProperties.forEach((oldVal, prop) => {
```

Comment:
This seems like a better place for this given `_handleChange` wouldn't have been called if values were set via JS. 👍